docs/en/guides/pipeline-integrations/getting-started/advanced_config.md

```diff
@@ -63,14 +63,28 @@ In this guide, you will use the Wizard in {% include product %} Desktop to gener
 
 ![New project](./images/advanced_config/2_new_project.png)
 
-**Step 2:** Launch the {% include product %} Desktop app and log in using the username and password from your {% include product %} site.
+**Step 2:** Launch the {% include product %} Desktop app and log in using the {% include product %} Application Session Launcher.
 
-![Login](./images/advanced_config/3_login.png)
+Upon launching Desktop and applications, you will be prompted to sign in to your {% include product %} site (example: `https://mysite.shotgrid.autodesk.com`). 
 
-![Load project](./images/advanced_config/4_load_project.png)
+![sign in](./images/sa-integrations-user-guide-session-launcher-01.png)
```

Comment:
These images don't show up http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/5feb147acfb7e23bd6f78537a74b2b0fb83351b4/5d83a936/?title=Configuration+Setup

Reply:
Thanks @julien-lang ! Let's see how the images look in this next build.